mirascope/core/vertex/call_response.py

```diff
@@ -0,0 +1,177 @@
+"""This module contains the `VertexCallResponse` class.
+
+usage docs: learn/calls.md#handling-responses
+"""
+
+from google.cloud.aiplatform_v1beta1.types import (
+    tool as gapic_tool_types,
+)
+from pydantic import computed_field
+from vertexai.generative_models import (
+    Content,
+    GenerationResponse,
+    Tool,
+)
+
+from ..base import BaseCallResponse
+from ._utils import calculate_cost
+from .call_params import VertexCallParams
+from .dynamic_config import VertexDynamicConfig
+from .tool import VertexTool
+
+
+class VertexCallResponse(
+    BaseCallResponse[
+        GenerationResponse,
+        VertexTool,
+        Tool,
+        VertexDynamicConfig,
+        Content,
+        VertexCallParams,
+        Content,
+    ]
+):
+    """Convenience wrapper around Vertex's `GenerateContentResponse`.
+
+    When using Mirascope's convenience wrappers to interact with Vertex models via
+    `VertexCall`, responses using `VertexCall.call()` will return a
```

Comment:
hmm, I'm noticing this is wrong.

Please reference the `openai` module equivalent that I've confirmed is correct and update here (and anywhere else, such as the `gemini` module) to match the correct format

(this is old from v0, not sure how I missed this)

Reply:
I fixed.